test-apps/display-test-app/package.json

```diff
@@ -12,26 +12,26 @@
   },
   "private": true,
   "scripts": {
-    "build": "npm run -s build:backend && npm run -s build:frontend && npm run -s build:frontend-webpack",
+    "build": "npm run -s build:backend && npm run -s build:frontend-vite",
     "build:backend": "tsc -p tsconfig.backend.json",
     "build:backend-webpack": "backend-webpack-tools build -s ./lib/backend/DtaElectronMain.js -o ./lib/build/",
     "build:frontend": "tsc",
-    "build:frontend-webpack": "cross-env DISABLE_TERSER=true DISABLE_NEW_JSX_TRANSFORM=true SKIP_PREFLIGHT_CHECK=true DISABLE_ESLINT_PLUGIN=true TRANSPILE_DEPS=false USE_FAST_SASS=true react-scripts --max_old_space_size=8192 build",
-    "build:webpack": "npm run build:frontend-webpack && npm run build:backend-webpack",
+    "build:frontend-vite": "cross-env NODE_OPTIONS=--max_old_space_size=8192 vite build",
```

Comment:
do we need to set `max_old_space_size`?
we did with react-scripts but idk if necessary for vite

Reply:
Yes, we still need to set them - else we would get a heap limit allocation error when trying to build
```
<--- JS stacktrace --->

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF684D507BF node_api_throw_syntax_error+175823
 2: 00007FF684CD5796 DSA_meth_get_flags+59654
 3: 00007FF684CD7480 DSA_meth_get_flags+67056
 4: 00007FF68577DCC4 v8::Isolate::ReportExternalAllocationLimitReached+116
 5: 00007FF685769052 v8::Isolate::Exit+674
 6: 00007FF6855EAF0C v8::internal::EmbedderStackStateScope::ExplicitScopeForTesting+124
 7: 00007FF6855E812B v8::internal::Heap::CollectGarbage+3963
 8: 00007FF6855FE363 v8::internal::HeapAllocator::AllocateRawWithLightRetrySlowPath+2099
 9: 00007FF6855FEC0D v8::internal::HeapAllocator::AllocateRawWithRetryOrFailSlowPath+93
10: 00007FF68560E3D0 v8::internal::Factory::NewFillerObject+816
11: 00007FF6852FF315 v8::internal::DateCache::Weekday+1349
12: 00007FF68581B1F1 v8::internal::SetupIsolateDelegate::SetupHeap+558193
13: 00007FF605AE079A
```